.scalafmt.conf

```diff
@@ -0,0 +1,20 @@
+style = defaultWithAlign
+maxColumn = 120
```

Comment:
What happens if we try 80? Or 100, at max?

Reply:
I just tried it with 80, I like it more too :)